rest/doc_api.go

```diff
@@ -277,14 +277,16 @@ func (h *handler) handlePutDoc() error {
 	var newRev string
 	var ok bool
 
+	roundTrip := h.getBoolQuery("roundtrip")
+
 	if h.getQuery("new_edits") != "false" {
 		// Regular PUT:
 		if oldRev := h.getQuery("rev"); oldRev != "" {
 			body[db.BodyRev] = oldRev
 		} else if ifMatch := h.rq.Header.Get("If-Match"); ifMatch != "" {
 			body[db.BodyRev] = ifMatch
 		}
-		newRev, err = h.db.Put(docid, body)
+		newRev, err = h.db.PutRoundTrip(docid, body, roundTrip)
```

Comment:
Review whether it makes more sense to include roundTrip as a flag in the default Put/PutExistingRev, instead of maintaining separate functions.